### PR TITLE
fix missing update-notifier issue

### PR DESCRIPTION
**Pull request by deancn**
## What/Why/How?
build fail due to missing `update-notifier` in package.json.

## Check yourself

- [ ] Code is linted
- [ ] Tested
- [ ] All new/updated code is covered with tests

### CODE REVIEW

**Comment by deancn** [comment]
@Oprysk  kindly help to merge, thank you.

**Comment by AlexVarchuk** [comment]
Hi, @deancn can you provide steps to reproduce the bug?

File: package.json [hunk 1]
```diff
@@ -161,6 +161,7 @@
     "stickyfill": "^1.1.1",
     "style-loader": "^3.3.1",
     "swagger2openapi": "^7.0.6",
+    "update-notifier": "^6.0.2",
```
Comment (AlexVarchuk) [inline package.json:164]: Why we should add this package to `redoc`? We use `update-notifier` in `redoc-cli`.

**Review by AlexVarchuk — CHANGES_REQUESTED** [review]

**Comment by deancn** [comment]
Hi @AlexVarchuk 

```bash
git clone https://github.com/Redocly/redoc
cd redoc/
yarn install
yarn run build:demo
```

The build error msg as follow:
```js
ERROR in cli/index.ts:4:33
TS2307: Cannot find module 'update-notifier' or its corresponding type declarations.
    2 | /* tslint:disable:no-implicit-dependencies */
    3 | import * as React from 'react';
  > 4 | import * as updateNotifier from 'update-notifier';
      |                                 ^^^^^^^^^^^^^^^^^
    5 | import { renderToString } from 'react-dom/server';
    6 | import { ServerStyleSheet } from 'styled-components';
    7 |

webpack 5.74.0 compiled with 1 error in 6116 ms
```

**Comment by AlexVarchuk** [comment]
You also should make install on the `cli` folder. Try to do
```
git clone https://github.com/Redocly/redoc
cd redoc/
yarn install
cd cli/
yarn install
yarn run build:demo
```

**Comment by deancn** [comment]
yes, it works.  Can also merge the PR?

**Comment by deancn** [comment]
Both are ok for me.  Up to you.

**Pull request closed without merging**